op-service/txmgr/txmgr_test.go

```diff
@@ -838,6 +838,14 @@ func TestIncreaseGasPrice(t *testing.T) {
 		name string
 		run  func(t *testing.T)
 	}{
+		{
+			name: "bump at least 1",
+			run: func(t *testing.T) {
+				tx, newTx := doGasPriceIncrease(t, 1, 3, 1, 1)
+				require.True(t, newTx.GasFeeCap().Cmp(tx.GasFeeCap()) > 0, "new tx fee cap must be larger")
+				require.True(t, newTx.GasTipCap().Cmp(tx.GasTipCap()) > 0, "new tx tip must be larger")
```

Comment:
Actually does `require.GreaterThan` work for big Ints?

Reply:
It does not. Always dreamed about writing a nice extension of testify for `big.Int` :)

Reply:
I feel like I've been round this loop of disappointment a few times before...